src/tests/ftest/util/dmg_utils_params.py

```diff
@@ -32,7 +32,12 @@ class DmgTransportCredentials(TransportCredentials):
     def __init__(self):
         """Initialize a TransportConfig object."""
         super(DmgTransportCredentials, self).__init__(
-            "/run/dmg/transport_config/*", "transport_config")
+            "/run/transport_config/*", "transport_config")
+
+        self.allow_insecure = BasicParameter(False, False)
+        self.ca_cert = BasicParameter(None, "./daosCA/certs/daosCA.crt")
+        self.cert = BasicParameter(None, "./daosCA/certs/admin.crt")
+        self.key = BasicParameter(None, "./daosCA/certs/admin.key")
```

Comment:
Sorry I missed this something earlier.  If now I'm seeing this correctly there are different values for `cert` and `key` depending on if we are setting up the `DaosServerTransportCredentials`, `DaosAgentTransportCredentials`, or the `DmgTransportCredentials`.  In other words:

- For `daos_server` the 'server' files are used:
```
        self.cert = BasicParameter(None, "./daosCA/certs/server.crt")
        self.key = BasicParameter(None, "./daosCA/certs/server.key")
```
- For `daos_agent` the 'admin' files are used:
```
        self.cert = BasicParameter(None, "./daosCA/certs/agent.crt")
        self.key = BasicParameter(None, "./daosCA/certs/agent.key")
```
- For `dmg` the 'admin' files are used:
```
        self.cert = BasicParameter(None, "./daosCA/certs/admin.crt")
        self.key = BasicParameter(None, "./daosCA/certs/admin.key")
```

The `allow_insecure` and `ca_cert` do appear to be common thought, correct?

Assuming this is true, I'll amend my previous comments and say that the all three `DaosServerTransportCredentials`, `DaosAgentTransportCredentials`, or the `DmgTransportCredentials classes should remain, but their `__inti__()` methods should only define unique default values for the `cert` and `key` values - even then they should probably use `BasicParameter.update_default(<value>)` after the `super().__init__()`.  The defaults for the common `allow_insecure` and `ca_cert` are better off defined in `TransportCredentials.__init__()`.

Reply:
Keep all the credential information on server, agent and dmg in on place would be easier for later maintenance and debug, rather than break into 2 places. 
ie:   separate 
        self.allow_insecure = BasicParameter(False, False)
        self.ca_cert = BasicParameter(None, "./daosCA/certs/daosCA.crt")     
and
        self.cert = BasicParameter(None, "./daosCA/certs/server.crt")
        self.key = BasicParameter(None, "./daosCA/certs/server.key")

Reply:
(2)gen_certificates.sh only run once not on all the hosts.
(3)When transport_config defined in the test.yaml, launch.py will NOT replace the allow_insecure value, even a different -sec mode is specified in the launch.py.